pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1925,6 +1952,35 @@ fn basic_fuzzing() {
                 ));
             }
 
+            fn check_locked_assets(
+                locked_assets: &HashMap<(IdentityId, Ticker), i32>,
+                tickers: &Vec<Ticker>,
+                users: &Vec<User>,
+            ) {
+                for ((did, ticker), balance) in locked_assets {
+                    assert_eq!(
+                        Portfolio::locked_assets(PortfolioId::default_portfolio(*did), ticker),
+                        *balance as u128
+                    );
+                }
```

Comment:
The for loop below will go over all tickers and users, so this for loop shouldn't be needed.

I recommend removing this for loop.